### DISCUSSION TITLE
[Request] 希望语音合成服务可以设置voice，发音角色

### DISCUSSION THREAD

**Discussion by burugo** [lobehub/lobehub#5157] [General Question] [answered]
### 🥰 需求描述

目前语音服务的语音合成只能选择模型，不能自定义voice，我使用one-api中转edge-tts的api，会报错
<img width="675" alt="image" src="https://github.com/user-attachments/assets/c06ea54e-f49e-456e-936d-ce5eb1df68b6" />
错误如下： [Error]: 500 Invalid voice 'alloy'.

### 🧐 解决方案

能设置voice参数就可以，因为edge和openai的语音角色名称不一样

[a comment by an automated account is not shown]

**Comment by SpeedupMaster** [comment, marked as answer]
![图片](https://github.com/user-attachments/assets/0abaa8a6-0533-4659-a3cc-c229902b08c2)
这里可以设置

**Reply by burugo** [reply]
看到了 谢谢

[a comment by an automated account is not shown]
